### DISCUSSION TITLE
Failed to load main.js?

### DISCUSSION THREAD

**Discussion by nintenfando** [DxltaMath/DMIx#23] [Q&A] [answered]
Not sure what happened but it isn't working. Here's the error message:
![Screenshot 2024-04-12 080315](https://github.com/DxltaMath/DMIx/assets/152813093/84238485-58d8-4fd9-a9b2-fb86a587de77)

**Comment by afkvido** [comment, marked as answer]
this project is old/unmaintained, sorry, if you know how to code, join us :)

**Reply by Joniecheese** [reply]
I am having this same issue, what should I do?
